File: old/src/jsMain/kotlin/org/baaahs/views/index/components/Events.kt
package org.baaahs.views.index.components

import csstype.Color
import csstype.integer
import mui.material.Box
import mui.material.Typography
import mui.material.TypographyAlign
import mui.material.styles.TypographyVariant
import mui.system.sx
import org.baaahs.blocks.blog.HorizontallyAlignedBlogCardWithShapedImage
import react.FC
import react.Props

val Events = FC<EventsProps> {
    Box {
        Typography {
            variant = TypographyVariant.h4
            sx {
                color = Color("text.primary")
                fontWeight = integer(700)
            }
            align = TypographyAlign.center
            gutterBottom = true

            +"upcoming events"

            HorizontallyAlignedBlogCardWithShapedImage {}
        }
    }
}

external interface EventsProps : Props {}

File: src/views/IndexView/components/Events/Events.js
import React from 'react';
import Box from '@mui/material/Box';
import Typography from '@mui/material/Typography';
//import Button from '@mui/material/Button';
//import useMediaQuery from '@mui/material/useMediaQuery';
//import { useTheme } from '@mui/material/styles';
import { HorizontallyAlignedBlogCardWithShapedImage } from 'blocks/blog';

const Events = () => {
  //const theme = useTheme();
  /*const isMd = useMediaQuery(theme.breakpoints.up('md'), {
    defaultMatches: true,
  });*/

  return (
    <Box>
      <Typography
        variant="h4"
        color="text.primary"
        align={'center'}
        gutterBottom
        sx={{
          fontWeight: 700,
        }}
        fontFamily={'Smooth Circulars'}
      >
        Upcoming Events
      </Typography>
      <HorizontallyAlignedBlogCardWithShapedImage
        data={{
          image:
            '/images/resized/campout_logo_promo.webp',
          description:
            'A campout put on by BAAAHS and friends over July 4th weekend. We’ve been putting it on for four years now and excited for our fifth! Our campouts are known for being whimsical, community driven, silly, and of course a whole lot of queer fun. We aim to create an awesome space for self-expression, creativity and kindness. Live music, sparkle-tastic lights, dance parties, group events, art and other surprises included. Eccentric outfits, gifting and showing off the best of you is strongly encouraged. And much like Burning Man, it is a Leave No Trace event.',
          title: 'BAAAHS Campout 2025: SideQuest?',
          date: '2–5 Jul',
          link: '/campout/register',
        }}
      ></HorizontallyAlignedBlogCardWithShapedImage>
    </Box>
  );
};

export default Events;
